omnigibson/systems/system_base.py

```diff
@@ -149,20 +94,21 @@ def reset(cls):
         """
         pass
 
+    @classmethod
+    def get_active_systems(cls):
+        """
+        Returns:
+            dict: Mapping from system name to system for all systems that are subclasses of this system AND active (initialized)
+        """
+        return {system.name: system for system in REGISTERED_SYSTEMS.values() if issubclass(system, cls) and system.initialized}
```

Comment:
shouldn't we just directly query from og.sim.scene.system_registry? (and of course wrap in in the if scene exists, initialized, etc...)

Reply:
done